release-notes/rn-6.16.0.md

```diff
@@ -0,0 +1,95 @@
+# Eclipse Che 6.16 Release Notes
+
+////
+
+Heading for blog post
+
+## $HIGHLIGHT1 $HIGHLIGHT2 $HIGHLIGHT3
+
+////
+
+The Eclipse Che 6.16 release contains the following notable features:
+
+* Added an ability to run Che Server with K8s/OS infrastructure on https with a self-signed certificate
+* $HIGHLIGHT2
+* $HIGHLIGHT3
+
+---
+
+Intro paragraph
+
+---
+
+## Quick Start
+
+Che is a cloud IDE and containerized workspace server - get started on:
+
+* Kubernetes ([single-user](https://www.eclipse.org/che/docs/kubernetes-single-user.html) or [multi-user](https://www.eclipse.org/che/docs/kubernetes-multi-user.html))
+* OpenShift ([single-user](https://www.eclipse.org/che/docs/openshift-single-user.html) or [multi-user](https://www.eclipse.org/che/docs/openshift-multi-user.html))
+* Docker ([single-user](https://www.eclipse.org/che/docs/docker-single-user.html) or [multi-user](https://www.eclipse.org/che/docs/docker-multi-user.html))
+* Try Eclipse Che live at [https://che.openshift.io](https://che.openshift.io)
+
+Learn more in our [documentation](https://www.eclipse.org/che/docs/infra-support.html) and start using a shared Che server or local instance today.
+
+---
+
+## An ability to run Che Server with K8s/OS infrastructure on https with a self-signed certificate
+
+![Che_on_https](https://www.eclipse.org/che/docs/images/workspaces/chrome_cert.png)
+
```

Comment:
I suggest adding a description of why this is a good thing.

"Security is increasingly important for web services, and many Eclipse Che users would like to ensure communication between the Che server and client is encrypted and authenticated. In previous versions, it was not possible to start workspaces in Eclipse Che if your container platform was using a self-signed certificate. In this release, we have added that ability. The Che administrator can enable this feature by creating a secret with the certificate, and configuring the server to ensure that the certificate is used by the Che server.

For more information, see  https://www.eclipse.org/che/docs/che-6/openshift-config.html#https-mode---self-signed-certs"

I can't really tell from the docs and from this description is this is an easy feature to use - the docs say that I have to enable the certificate for each workspace, but I don't know how to do that. Would it be possible (in the docs) to clarify how to use this feature?

Reply:
> I suggest adding a description of why this is a good thing.

Done.

> I can't really tell from the docs and from this description is this is an easy feature to use - the docs say that I have to enable the certificate for each workspace, but I don't know how to do that. Would it be possible (in the docs) to clarify how to use this feature?

I think https://www.eclipse.org/che/docs/che-6/openshift-config.html#https-mode---self-signed-certs is not up to date. Have you seen Eugene's PR? Maybe things are clearer there https://github.com/eclipse/che-docs/pull/630/files
Please notify me if changes according to this comment are still needed.